test/drivers/second_quantization/test_molecule_driver.py

```diff
@@ -60,14 +62,16 @@ def test_invalid_kwarg(self):
         (ElectronicStructureDriverType.PYQUANTE, -1.1169989925292956),
         (ElectronicStructureDriverType.GAUSSIAN, -1.1169989967389082),
     )
-    @requires_extra_library
     def test_driver(self, config):
         """test driver"""
         driver_type, hf_energy = config
         driver = ElectronicStructureMoleculeDriver(
             self._molecule, basis="sto3g", driver_type=driver_type
         )
-        driver_result = driver.run()
+        try:
+            driver_result = driver.run()
+        except MissingOptionalLibraryError as ex:
```

Comment:
I guess this relates to my comment above: how can this still occur?
Is it because we defer the lazy loading of pyscf into another class?